### ISSUE TITLE
Negative numbers aren't plotted in right order

### ISSUE THREAD

**Issue by owainevans** [probmods/webppl-viz#13]
Plotting discrete erp with positive and negative values:

``` javascript
viz.vegaPrint(ParticleFilter(function() {
  return {x: uniformDraw([-10,-20,0,10,20])};
},500))
```

**Comment by longouyang** [comment]
Fixed in 63e58b296d76b5ef27c8949484d4c91e6ad2dbbb, I believe (lemme know if that doesn't work for you)

**Comment by owainevans** [comment]
yep, works for me!
